dist/index.js

```diff
@@ -0,0 +1,82 @@
+class BookStore {
+  constructor() {
+    this.books = [];
+  }
+
+  addBook(title, author) {
+    const book = {
+      title: title,
+      author: author
+    };
+    this.books.push(book);
+    return book;
+  }
+
+  removeBook(index) {
+    this.books.splice(index, 1);
```

Comment:
- [OPTIONAL] I recommend that you use the `filter` array method instead of `splice` because `filter` does not mutate the array but returns a new one, you can use it like this `this.books = this.books.filter(book => book.id != id)`.
Not mutating an array is a great practice and it will increase your performance